Features/ContentActions/MoveContent.feature

```diff
@@ -18,12 +18,12 @@ Feature: Move content
         And I select the "eZ Platform/Older News" folder in the Universal Discovery Widget
         And I confirm the selection
         Then I am notified that "News Flash" has been moved under "Older News"
-        And I see "Older News/News Flash" in the content tree
-        And I do not see "Origin/News flash" in the content tree
+        And There is a content "Older News/News Flash"
```

Comment:
I'd even add alternatives to those sentences, even if the implementation is identical:
- `the content item "Older News/Tomorrow news/News Flash" was (not) removed`
- `the content item "Older News/Tomorrow news/News Flash" was (not) sent to trash`
- `the content item was moved to "Older News/Tomorrow news/News Flash"`

They can all use the same `theContentItemExists()` implementation, but will sound better in the scenarii.